packages/@sanity/base/src/change-indicators/overlay/Connector.styled.tsx

```diff
@@ -0,0 +1,87 @@
+import {Theme} from '@sanity/ui'
+import styled, {css} from 'styled-components'
+import {ClampedRect} from './ClampedRect'
+
+interface PathInterface {
+  focused: boolean
+  revertedHovered: boolean
+  hovered: boolean
+  theme: Theme
+}
+
+export const DebugRectWrapper = styled.rect`
```

Comment:
Since it's not strictly _wrapping_ anything, but rather styling it, maybe remove the "*Wrapper" suffix?

Reply:
For styled components I would disagree, my take on it is that the goal with the naming with suffix (in this case `Wrapper`) should be used for easy identification of what is a styled component and what isn't when opening a component (in this example `Connector`). As for the `Wrapper` word itself, I think it's fitting because all the styled components are doing is wrapping actual elements with a new element atop (`rect`, `path`, `sanity-ui` components, etc) :)